emu_mps/mps_config.py

```diff
@@ -43,6 +47,8 @@ def __init__(
         max_krylov_dim: int = 100,
         extra_krylov_tolerance: float = 1e-3,
         num_gpus_to_use: int = DEVICE_COUNT,
+        autosave_prefix: str = "emu_mps_save_",
+        autosave_dt: int = 60,
```

Comment:
should we default this to `3600` i.e. an hour?

Reply:
I think more often (10 minutes) is better but the logs will be annoying for very long simulations
What about writing a single log at the beginning? And/or moving the warning log on autosave to a debug/info log

Reply:
The reason why 10 minutes is that I don't think there's a reason for longer, since 
1. pickle is quick
2. if pickle takes a bit of time, it's because there is a lot of data (many gbs), which happens when bond dim is high and simulation steps take hours, therefore it wouldn't happen every 10 minutes
3. outside of crashes, I personally get sometimes problem with internet connection dropping and cluster ssh disconnecting when running simulations in interactive fashion, and waiting an hour can be already quite long (although in that case it's better to run sbatch)